src/main/java/com/stripe/exception/StripeException.java

```diff
@@ -1,8 +1,17 @@
 package com.stripe.exception;
 
+import com.stripe.model.StripeError;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
 public abstract class StripeException extends Exception {
   private static final long serialVersionUID = 2L;
 
+  @Setter
+  StripeError error;
```

Comment:
Can you clarify why this approach is better than making `StripeError` inherit from `Exception` and have other exceptions inherit from it?

Reply:
I think it's better to keep the two classes separate. `StripeError` is the model class that represents the API resource defined as `api_errors` in the OpenAPI spec. `StripeException` is the exception that is raised when an API request fails. The two are related, but they serve different purposes.